src/core/api/index.js

```diff
@@ -45,19 +59,29 @@ export function callApi({
   const apiURL = `${API_BASE}/${endpoint}/${queryString}`;
 
   return fetch(apiURL, options)
-    .then((response) => {
+    .then((response) => response.json().then(
```

Comment:
Anyway, promises are dumb and hard to read. If we switched to async/await it would be a try/catch block and it would be a lot easier to read.